File: src/main/java/com/cag/cagbackendapi/dtos/UserDto.java
package com.cag.cagbackendapi.dtos;

import lombok.Data;
import java.util.UUID;

@Data
public class UserDto {
    private UUID id;
    private String name;
    private String email;

    public String toString() {
        return "name: " + this.name + ", email: " + this.email;
    }
}

File: src/main/java/com/cag/cagbackendapi/dtos/UserDto.kt
package com.cag.cagbackendapi.dtos;

import java.util.UUID

data class UserDto(
    var userId: UUID?,
    var first_name: String?,
    var last_name: String?,
    var email: String?,
    var active_status: Boolean?,
    var session_id: String?,
    var img_url: String?,
    var agreed_18: Boolean?,
    var agreed_privacy: Boolean?)
